programs/svm-spoke/src/instructions/deposit.rs

```diff
@@ -17,12 +17,7 @@ use crate::{
     output_token: Pubkey,
     input_amount: u64,
     output_amount: u64,
-    destination_chain_id: u64, // TODO: we can remove some of these instructions props
```

Comment:
we dont need the extra fields here in the instructions. also, be removing them, we could re-use this `DepositV3` context for the `depositNow` method.

Reply:
Originally before removing the extra instructions, I was not able to re-call deposit_v3. I've now tried again and with the removed extra instructions (in particular, the ones missing when calling deposit_v3_now) I was able to re-use the original deposit_v3 method. take a look again.